### DISCUSSION TITLE
Question: ListGroup with Multiselection

### DISCUSSION THREAD

**Discussion by samuelhurni** [Megabit/Blazorise#4008] [Q&A]
Hello together

I want to ask if there is a way to use a ListGroup Element with Multiselect feature.

I know about the Datagrid possibilities but I want to use a Card with multiple rows, so for this a List Element is much better. But I need also the multiselect feature, so have I do this byself or is there an easy way with blazorise?

Regards Samuel

**Comment by stsrki** [comment]
Not possible at the moment but we can plan it for the future milestone.

@David-Moreira
